src/relay/bin/commander/__tests__/options.test.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+let fetchSchemaOptions;
+let relayCompilerOptions;
+beforeEach(() => {
+  jest.isolateModules(() => {
+    fetchSchemaOptions = require('../options').fetchSchemaOptions;
+    relayCompilerOptions = require('../options').relayCompilerOptions;
+  });
+});
+
+describe('fetch schema', () => {
+  it('no args', () => {
+    expect(fetchSchemaOptions(['node', 'test'])).toMatchInlineSnapshot(`
+      Object {
+        "filename": "schema.graphql",
+      }
```

Comment:
Why use snapshot here instead of just `toEqual({filename: 'schema.graphql'})`?

Reply:
Because I am often looking for lazy solutions but using the equality tests is obviously better. Please have a look again.